ChangeLog.d/fix-pkgconfig-static-linking-windows.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix:
+   * Fix static linking with pkgconfig file in Windows.
+     This adds bcrypt library in linker flag for mbedcrypto.
+     bcrypt library is required for BCryptGenRandom function
+     used in entropy_poll.c file.
```

Comment:
There's too much information here. Changelog entries are for users, not for developers, so they shouldn't go into details about the change. Also, we do insist on grammatically correct English in changelog entries. Suggestion:

> Fix pkgconfig files on Windows: the bcrypt library was missing when linking statically.

Reply:
Thank you for the suggestion. I have added it in the changelog file.